src/styles/_map.scss

```diff
@@ -19,3 +19,9 @@
     display: none;
   }
 }
+
+@media (min-width: 641px) {
+  .leaflet-control {
+    top: 40px;
```

Comment:
This is pushing down the zoom buttons below the viewport:

![image](https://user-images.githubusercontent.com/161748/83872825-94a94980-a6e7-11ea-85c2-ebbad7ebfa90.png)

Reply:
Shrunk the size of the search bar so there's no reason to move it down as I did.